xmantissa/websession.py

```diff
@@ -277,16 +356,23 @@ def cbLoginSuccess(input):
                     self.savorSessionCookie(request)
             return input
 
-        return guard.SessionWrapper.login(self, request, session, creds, segments
-                                          ).addCallback(cbLoginSuccess)
+        return (
+            guard.SessionWrapper.login(
+                self, request, session, creds, segments)
+            .addCallback(cbLoginSuccess))
 
 
     def explicitLogout(self, session):
```

Comment:
[Codecov confirms that it's not covered](https://codecov.io/github/twisted/mantissa/xmantissa/websession.py?ref=b103aa11b20ffe6667e6a0eb0e1df5fa04168708#l-377). I didn't change the code in this branch, but I guess I may as well add a test while I'm here.